tools/ansible/ansible.cfg

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
```

Comment:
The copyright on the new files needs to match what we're using elsewhere in this file; to adhere to the Apache 2.0 requirements, which is different than what we use for our internal repositories.